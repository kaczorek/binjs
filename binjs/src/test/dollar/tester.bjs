#!/bin/js
#
# \0 is not possible to double escape into bash
#
# There are various methods in colour.js to show how to print colours
#
var i = 0;
for(; i < 128; i++) {
	$.setEnv("i", i);

	echo "\033[${i}m ${i}:colour test \033[0m"

}
