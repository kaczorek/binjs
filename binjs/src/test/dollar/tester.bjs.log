\033[0m 0:colour test \033[0m
\033[1m 1:colour test \033[0m
\033[2m 2:colour test \033[0m
\033[3m 3:colour test \033[0m
\033[4m 4:colour test \033[0m
\033[5m 5:colour test \033[0m
\033[6m 6:colour test \033[0m
\033[7m 7:colour test \033[0m
\033[8m 8:colour test \033[0m
\033[9m 9:colour test \033[0m
\033[10m 10:colour test \033[0m
\033[11m 11:colour test \033[0m
\033[12m 12:colour test \033[0m
\033[13m 13:colour test \033[0m
\033[14m 14:colour test \033[0m
\033[15m 15:colour test \033[0m
\033[16m 16:colour test \033[0m
\033[17m 17:colour test \033[0m
\033[18m 18:colour test \033[0m
\033[19m 19:colour test \033[0m
\033[20m 20:colour test \033[0m
\033[21m 21:colour test \033[0m
\033[22m 22:colour test \033[0m
\033[23m 23:colour test \033[0m
\033[24m 24:colour test \033[0m
\033[25m 25:colour test \033[0m
\033[26m 26:colour test \033[0m
\033[27m 27:colour test \033[0m
\033[28m 28:colour test \033[0m
\033[29m 29:colour test \033[0m
\033[30m 30:colour test \033[0m
\033[31m 31:colour test \033[0m
\033[32m 32:colour test \033[0m
\033[33m 33:colour test \033[0m
\033[34m 34:colour test \033[0m
\033[35m 35:colour test \033[0m
\033[36m 36:colour test \033[0m
\033[37m 37:colour test \033[0m
\033[38m 38:colour test \033[0m
\033[39m 39:colour test \033[0m
\033[40m 40:colour test \033[0m
\033[41m 41:colour test \033[0m
\033[42m 42:colour test \033[0m
\033[43m 43:colour test \033[0m
\033[44m 44:colour test \033[0m
\033[45m 45:colour test \033[0m
\033[46m 46:colour test \033[0m
\033[47m 47:colour test \033[0m
\033[48m 48:colour test \033[0m
\033[49m 49:colour test \033[0m
\033[50m 50:colour test \033[0m
\033[51m 51:colour test \033[0m
\033[52m 52:colour test \033[0m
\033[53m 53:colour test \033[0m
\033[54m 54:colour test \033[0m
\033[55m 55:colour test \033[0m
\033[56m 56:colour test \033[0m
\033[57m 57:colour test \033[0m
\033[58m 58:colour test \033[0m
\033[59m 59:colour test \033[0m
\033[60m 60:colour test \033[0m
\033[61m 61:colour test \033[0m
\033[62m 62:colour test \033[0m
\033[63m 63:colour test \033[0m
\033[64m 64:colour test \033[0m
\033[65m 65:colour test \033[0m
\033[66m 66:colour test \033[0m
\033[67m 67:colour test \033[0m
\033[68m 68:colour test \033[0m
\033[69m 69:colour test \033[0m
\033[70m 70:colour test \033[0m
\033[71m 71:colour test \033[0m
\033[72m 72:colour test \033[0m
\033[73m 73:colour test \033[0m
\033[74m 74:colour test \033[0m
\033[75m 75:colour test \033[0m
\033[76m 76:colour test \033[0m
\033[77m 77:colour test \033[0m
\033[78m 78:colour test \033[0m
\033[79m 79:colour test \033[0m
\033[80m 80:colour test \033[0m
\033[81m 81:colour test \033[0m
\033[82m 82:colour test \033[0m
\033[83m 83:colour test \033[0m
\033[84m 84:colour test \033[0m
\033[85m 85:colour test \033[0m
\033[86m 86:colour test \033[0m
\033[87m 87:colour test \033[0m
\033[88m 88:colour test \033[0m
\033[89m 89:colour test \033[0m
\033[90m 90:colour test \033[0m
\033[91m 91:colour test \033[0m
\033[92m 92:colour test \033[0m
\033[93m 93:colour test \033[0m
\033[94m 94:colour test \033[0m
\033[95m 95:colour test \033[0m
\033[96m 96:colour test \033[0m
\033[97m 97:colour test \033[0m
\033[98m 98:colour test \033[0m
\033[99m 99:colour test \033[0m
\033[100m 100:colour test \033[0m
\033[101m 101:colour test \033[0m
\033[102m 102:colour test \033[0m
\033[103m 103:colour test \033[0m
\033[104m 104:colour test \033[0m
\033[105m 105:colour test \033[0m
\033[106m 106:colour test \033[0m
\033[107m 107:colour test \033[0m
\033[108m 108:colour test \033[0m
\033[109m 109:colour test \033[0m
\033[110m 110:colour test \033[0m
\033[111m 111:colour test \033[0m
\033[112m 112:colour test \033[0m
\033[113m 113:colour test \033[0m
\033[114m 114:colour test \033[0m
\033[115m 115:colour test \033[0m
\033[116m 116:colour test \033[0m
\033[117m 117:colour test \033[0m
\033[118m 118:colour test \033[0m
\033[119m 119:colour test \033[0m
\033[120m 120:colour test \033[0m
\033[121m 121:colour test \033[0m
\033[122m 122:colour test \033[0m
\033[123m 123:colour test \033[0m
\033[124m 124:colour test \033[0m
\033[125m 125:colour test \033[0m
\033[126m 126:colour test \033[0m
\033[127m 127:colour test \033[0m
