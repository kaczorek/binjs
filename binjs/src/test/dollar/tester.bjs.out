binjs_import("~lib/binjs.js");

var i = 0;
for(; i < 128; i++) {
	$.setEnv("i", i);

	binjs_exec("echo \"\\033[${i}m ${i}:colour test \\033[0m\"");

}
;
	binjs_exec("echo \"   \033[${i}m ${i}:oioi \033[0m  \"  ");
}

